.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+staleLabel: stale
+daysUntilClose: 7
+
+pulls:
+    daysUntilStale: 180 # ~6 months
```

Comment:
ultimately I think this should be shorter, but since currently many prs are stalled because we haven't reviewed, 6 months seemed like a good starting point.